src/Unleash.php

```diff
@@ -130,4 +137,9 @@ protected function formatResponse($data): array
     {
         return Arr::get($data, 'features', []);
     }
+
+    protected function checkExtraOk($strategyData): bool
+    {
+        return true;
+    }
```

Comment:
What is this for?

Reply:
This is for checking extra data in the strategy. We used unleash in GitLab, its api returns response in a different format. For example, feature 'test_user_ids' looks like this:
```
{
    "name": "test_user_ids",
    "description": "You can designate user by ip, or by cookie.",
    "active": true,
    "version": "new_version_flag",
    "created_at": "2020-12-05T23:33:51.406Z",
    "updated_at": "2020-12-09T13:14:54.856Z",
    "strategies": [
        {
            "id": 7,
            "name": "userWithId",
            "parameters": {
                "userIds": "qing"
            },
            "scopes": [
                {
                    "id": 15,
                    "environment_scope": "test"
                },
                {
                    "id": 16,
                    "environment_scope": "local"
                }
            ]
        }
    ]
}
```
It is the scopes data within a strategy that we are checking. It is not really strategy specific, so I think it would be better to check it not within the Strategy class but just in Unleash. This could of course also be used for other unleash api's with other generic data.

Reply:
Ok, I think I understand. I think it could be clearer what this does so maybe this could be moved to it's own PR so we can reason about it a bit more?

Reply:
Yes, I agree. It's been removed. Thanks